server/terraform/modules/myhealth/main.tf

```diff
@@ -26,6 +26,7 @@ provider "google-beta" {
 # Google Project
 resource "google_project" "project" {
   provider        = google-beta
+  count           = var.create_project ? 1 : 0
```

Comment:
I wonder if terraform import can be used instead? in general, what is the plan for shared state management? remote_state on gcs?

Reply:
@matthewblain - please file an issue for this as it is important. For now, I'm the only one applying Terraform but that will change. I'm unclear if we should use Hashicorp or Google as the backend. It is easier to have a smaller number of 3rd parties and we clearly already use Google.

Terraform Docs for Remote Backend:
https://www.terraform.io/docs/state/remote.html

I'm experimenting with the import now.

Reply:
I guess this fits with the Terraform model: either a resource is managed by Terraform, or its not. And in this scneario, it is not.

Reply:
Filed: #1744